### PR TITLE
Clarifying the limitations of restoring from full versus partial snapshots

### PR DESCRIPTION
**Pull request by paigecalvert**
About Backup and Restore: https://deploy-preview-2355--replicated-docs-upgrade.netlify.app/enterprise/snapshots-understanding

Restoring from Backups: https://deploy-preview-2355--replicated-docs-upgrade.netlify.app/enterprise/snapshots-restoring-full

### CODE REVIEW

File: docs/enterprise/snapshots-understanding.mdx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+The following table lists which versions of Velero are compatible with each version of KOTS. For more information, see the [Velero documentation](https://velero.io/docs/).
+
+| KOTS version | Velero version |
+|------|-------------|
+| 1.15 to 1.20.2 | 1.2.0 |
+| 1.20.3 to 1.94.0 | 1.5.1 through 1.9.x |
+| 1.94.1 and later | 1.6.x through 1.12.x |
+
+## About Backups
```
Comment (paigecalvert) [inline docs/enterprise/snapshots-understanding.mdx:45]: Rewrote the description of backups to clarify the different between full and partial backups

File: docs/partials/snapshots/_restore-types.mdx [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Snapshots supports the following types of restores:
+* Restore both the application and the KOTS admin console to the existing cluster or to a new cluster (also referred to as a _full_ restore)
+* Restore the KOTS admin console to the existing cluster or to a new cluster
+* Restore the application to the existing cluster (also referred to as a _partial_ restore)
```
Comment (paigecalvert) [inline docs/partials/snapshots/_restore-types.mdx:4]: Added this partial that lists the different restore options.

File: docs/partials/snapshots/_restore-types.mdx [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Snapshots supports the following types of restores:
+* Restore both the application and the KOTS admin console (also referred to as a _full_ restore)
+* Restore the KOTS admin console only
+* Restore the application only (also referred to as a _partial_ restore)
```
Comment (paigecalvert) [inline docs/partials/snapshots/_restore-types.mdx:4]: ^ Added this partial to list the different types of restores

File: docs/partials/snapshots/_limitation-cli-restores.mdx [hunk 4]
```diff
@@ -0,0 +1 @@
+Only full backups can be restored using the kots CLI. To restore an application from a partial backup, use the admin console. See [Restore the Application Only Using the Admin Console](snapshots-restoring-full#admin-console).
```
Comment (paigecalvert) [inline docs/partials/snapshots/_limitation-cli-restores.mdx:1]: ^ Clarify that only full backups can be restored using the CLI

File: docs/partials/snapshots/_limitation-dr.mdx [hunk 5]
```diff
@@ -0,0 +1 @@
+Only full backups that include both the application and the admin console can be restored to a new cluster in disaster recovery scenarios. Partial backups that include the application only _cannot_ be restored to a new cluster, and are therefore not useable for disaster recovery.
```
Comment (paigecalvert) [inline docs/partials/snapshots/_limitation-dr.mdx:1]: ^ Clarify that only full backups can be restored to a new cluster in dr scenarios

File: docs/enterprise/snapshots-restoring-full.mdx [hunk 6]
```diff
@@ -1,82 +1,90 @@
...
 
-- kots CLI from a full backup
-- Admin console from either a full or partial backup
+<RestoreTypes/>
 
-For more information about the restore process, see [About Restores](snapshots-understanding#restores) in _About Backup and Restore_.
+You can do any type of restore from a full backup using the kots CLI. You can also restore an application from a full or partial backup using the admin console.
 
-## Restore from the CLI {#full-cli}
+## Limitations
```
Comment (paigecalvert) [inline docs/enterprise/snapshots-restoring-full.mdx:20]: Added a new Limitations section that lists the limitations related to restoring

File: docs/enterprise/snapshots-restoring-full.mdx [hunk 7]
```diff
@@ -1,82 +1,90 @@
...
+* <InstallMethod/>
+* <CliRestores/>
 
-Use the corresponding CLI procedure for your environment:
+For a full list of limitations and considerations related to the snapshots feature, see  [Limitations and Considerations](snapshots-understanding#limitations-and-considerations) in _About Backup and Restore_. 
 
-- [Existing Clusters](#existing)
-- [Online Embedded kURL Clusters](#online)
-- [Air Gapped Embedded kURL Clusters](#air-gapped)
+## Restore From a Full Backup Using the CLI {#full-cli}
```
Comment (paigecalvert) [inline docs/enterprise/snapshots-restoring-full.mdx:31]: Note: I didn't really review the content of these procedures for the purpose of this PR in favor of focusing on clarifying limitations of restores. I can do a follow up PR as needed to make any edits to these procedures too

File: docs/enterprise/snapshots-understanding.mdx [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+- Microsoft Azure
+- S3-Compatible
+- Network File System (NFS)
+- Host Path
+
+kURL installers that include the Velero add-on also include a locally-provisioned object store. By default, embedded kURL clusters are preconfigured in the admin console to store backups in the locally-provisioned object store. This object store is sufficient for only rollbacks and downgrades and is not a suitable configuration for disaster recovery. Replicated recommends that you configure a snapshots storage destination that is external to the cluster in the admin console for embedded kURL clusters.
+
+For information about how to configure backup storage destinations for snapshots, see the [Configuring Backup Storage](/enterprise/snapshots-velero-cli-installing) section.
+
+## About Restores {#restores}
```
Comment (paigecalvert) [inline docs/enterprise/snapshots-understanding.mdx:85]: Also rewrote this restores section to clarify the different types of restores

File: docs/enterprise/snapshots-restoring-full.mdx [hunk 9]
```diff
@@ -92,39 +100,32 @@ To restore a backup in an air gapped embedded kURL cluster:
...
-    For more information about the available restore options, including application only and admin console only, see [restore](/reference/kots-cli-restore-index/) in _Reference_.
+## Restore the Application Only Using the Admin Console {#admin-console}
 
-## Restore the Application from the Admin Console {#admin-console}
+You can restore an application from a full or partial backup using the admin console.
 
-You can restore the application only, known as a _partial restore_, from either a full backup or a partial backup in the admin console.
+### Restore an Application From a Full Backup
 
-Alternatively, you can use the kots CLI to restore any type of backup. See [Restore Any Backup from the CLI](#full-cli).
```
[the remark this answers is not in the document]
Comment (sgalsaleh) [reply docs/enterprise/snapshots-restoring-full.mdx:110]: It's not possible to restore partial backups from the CLI, but it is possible to only restore the application from a full backup using the CLI though. Is that captured in the docs anywhere?

File: docs/enterprise/snapshots-understanding.mdx [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+
+If you manage multiple applications with the admin console, data from all applications that support backups is included in a full backup. To be included in full backups, each application must include a manifest file with `kind: Backup` and `apiVersion: velero.io/v1`, which you can check for in the admin console.
+
+Full backups are recommended because they support all types of restores. For example, you can restore both the admin console and application from a full backup to a new cluster in disaster recovery scenarios. Or, you can use a full backup to restore only application data for the purpose of rolling back after deploying a new version of an application.
+
+### Application-Only (Partial) Backups
+
+Partial backups back up the application volumes and manifest files only. Partial backups do not back up the KOTS admin console.
+
+Partial backups can be useful if you need to roll back after deploying a new application version. Partial backups of the application only _cannot_ be restored to a new cluster, and are therefore are not useable for disaster recovery scenarios.
```
Comment (sgalsaleh) [inline docs/enterprise/snapshots-understanding.mdx:64]:
```suggestion
Partial backups can be useful if you need to roll back after deploying a new application version. Partial backups of the application only _cannot_ be restored to a new cluster, and are therefore not useable for disaster recovery scenarios.
```

File: docs/enterprise/snapshots-restoring-full.mdx (hunk 9 above)
[the remark this answers is not in the document]
Comment (sgalsaleh) [reply docs/enterprise/snapshots-restoring-full.mdx:110]: Just saw it capture below.

**Review by sgalsaleh — APPROVED** [review]

**Pull request merged**